gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -138,6 +142,39 @@ def _bigquery_to_vcf_shards(
          | vcfio.WriteVcfDataLines())
 
 
+def _get_schema(input_table):
+  # type: (str) -> bigqueryv2.TableSchema
+  project_id, dataset_id, table_id = bigquery_util.parse_table_reference(
+      input_table)
+  credentials = (client.GoogleCredentials.get_application_default().
+                 create_scoped(['https://www.googleapis.com/auth/bigquery']))
+  bigquery_client = bigquery_v2.BigqueryV2(credentials=credentials)
+  table = bigquery_client.tables.Get(bigquery_v2.BigqueryTablesGetRequest(
+      projectId=project_id, datasetId=dataset_id, tableId=table_id))
+  return table.schema
+
+
+def _extract_annotation_names(schema):
+  # type: (bigquery_v2.TableSchema) -> Dict[str, List[str]]
+  """Returns a mapping of annotation id to the corresponding annotation names.
+
+  The annotation names are useful for reconstructing the annotation str (e.g.,
```

Comment:
Please add a comment about the assumption of annotation names (i.e. they're a record within alternate bases; any such record is considered an annotation field).